docs/workflows/api-token.md

```diff
@@ -1,17 +1,14 @@
 ---
 title: Create an API token
 sidebar_position: 6
-# Display h2 to h2 headings
-toc_min_heading_level: 2
-toc_max_heading_level: 2
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Create an API Token
+# API Token Overview
 
-All [Beyond Identity API](https://developer.beyondidentity.com/api/v1) endpoints require authentication using an access token. The access token is generated through OAuth 2.0 or OIDC, using the authorization code flow or the client credentials flow.
+All [Beyond Identity API](https://developer.beyondidentity.com/api/v1) endpoints require authentication using an access token. The access token is generated through OAuth 2.0 or OIDC, using the authorization code flow or the client credentials flow. All access tokens are JWT. Two token types are supported: self-contained tokens as JWS where referential tokens as JWE. The default token on creation is self-contained.
```

Comment:
Referential vs self-contained is called `token format` in our terminology. I think it would be good to reflect that here.